.circleci/config.yml

```diff
@@ -135,3 +150,18 @@ workflows:
       - type check js (optional)
       - e2e tests
       - release if needed
+          filters:
```

Comment:
this probably needs to be part of `triggers` or a level lower?

Reply:
https://github.com/algolia/shipjs/blob/main/.circleci/config.yml#L90-L96

It works like that.
I hate that I need to find an example every time I modify circleci config :(